examples/test_method/Form_ExitStruct.rb

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+
+require 'rubygems'
+require "newt"
+
+begin
+
+  Newt::Screen.new
+
+  b1 = Newt::Button.new(1, 5, "Button1")
+  b2 = Newt::Button.new(1, 9, "Button2")
+
+  b = Newt::Button.new(1, 13, "Exit")
+
+  f = Newt::Form.new
+  f.add_hotkey(?\e.ord)
+  f.add(b1, b2, b)
+
+  rv = f.run()
+
+ensure
+  Newt::Screen.finish
+end
+
+puts rv.inspect
+puts "Button1" if rv == b1
+puts "Button2" if b2 == rv
+puts "Exit" if rv == b
+puts "Escape" if (rv.reason == Newt::EXIT_HOTKEY && rv.key == ?\e.ord)
```

Comment:
This is a change of essential interface, unfortunately we don't yet have any tests. While I won't insist on this, do you have an idea how to create some kind of integration test from this example? I was thinking about using tmux session, sending keys to it:

    tmux send-keys -t "${window_name}:${pane_number}" Tab Enter

If each test file returns zero, that's a pass. If it hangs, testing suite (shell script loop) kills it and marks as a failure.